flink-runtime/src/main/java/org/apache/flink/runtime/state/memory/MemoryStateBackend.java

```diff
@@ -65,8 +54,8 @@
  *
  * <pre>{@code
  * 		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
- * 		env.setStateBackend(new HashMapStateBackend());
- * 		env.getCheckpointConfig().setCheckpointStorage(new JobManagerCheckpointStorage());
+ * 		parameters.getEnv().setStateBackend(new HashMapStateBackend());
```

Comment:
Change to the comment looks wrong.

Reply:
hahaha, some find & replace that went wrong